mitodl/micromasters
# Store a smaller version of avatars and use on leaners page

#### What are the relevant tickets?
Fixes #2055
Fixes #1889

#### What's this PR do?
- Creates a new `image_small` field which stores a smaller version of the avatar
- Adds `image_small` to the serializer readonly fields and populates it when `image` is updated. `image_small` should not be updateable by itself
- Store it when indexing the profiles

#### How should this be manually tested?
- Make sure you have at least two users, one who is staff for a program and one who is a learner for that program. At least the learner should have an avatar.
- Run the migrations, then run `recreate_index`
- Log in as the staff user and go to `/learners` for the program and view the avatars. If you inspect the avatar image it should be `64x64` in size.
- Log in as the learner and update your avatar to something else
- Log in as the staff user again. Go to `/learners` and view the avatars. You should see the new avatar. If you inspect the image it should be `64x64` in size.
